webstart.ts

```diff
@@ -265,7 +254,6 @@ function webStart() {
     });
     editor.on("inputRead", function onChange(editor, input) {
       if (input.text[0] === ";" || input.text[0] === " " || input.text[0] === ":") {
-        isClassMethod = false;
```

Comment:
Sorry, on this line, I don't know why I deleted it, is it needed?

Reply:
Can we revert this change too? This variable is required for autocomplete logic and looks like you removed it by mistake

Reply:
its needed to ensure we can invoke the autocomplete with the right lists

Reply:
Got it ,i've added it back, sorry about that